elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideConfigProperties.java

```diff
@@ -43,6 +43,11 @@
      */
     private DynamicConfigProperties dynamicConfig;
 
+    /**
+     * Settings for the Aggregation Store.
+     */
+    private ControllerProperties aggregationStore;
```

Comment:
Instead of reusing ControllerProperties, we should create a new properties file.

Reply:
Sure.